pkgs/development/tools/build-managers/gnumake/default.nix

```diff
@@ -24,6 +25,8 @@ stdenv.mkDerivation rec {
 
   # to update apply these patches with `git am *.patch` to https://git.savannah.gnu.org/git/make.git
   patches = [
+    # See patch message.
+    ./make-cxx.patch
```

Comment:
```suggestion
    ./0000-make-cxx.patch
```

Non blocking

Reply:
Updated the patch set as suggested in the comment above `patches`, cleaned up that a bit to use separate patches directory with improved docs.